benchclients/python/benchclients/conbench.py

```diff
@@ -57,7 +57,11 @@ class ConbenchClient(RetryingHTTPClient):
 
     timeout_login_request = (3.5, 10)
 
-    def __init__(self, default_retry_for_seconds=None):
+    def __init__(self,
+                 url: Optional[str] = os.getenv("CONBENCH_URL"),
+                 email: Optional[str] = os.getenv("CONBENCH_EMAIL"),
+                 password: Optional[str] = os.getenv("CONBENCH_PASSWORD"),
+                 default_retry_for_seconds=None):
```

Comment:
Would you mind running `make lint` from the root of the repo? You'll need the packages in [requirements-dev.txt](https://github.com/conbench/conbench/blob/main/requirements-dev.txt).

Reply:
Hi @austin3dickey - I tried this, but it seemed to try to lint my virtual env directory (`.venv`) - I'm not sure how to do this properly.  Could you kindly let me know how best to do so?  Thanks.

Reply:
I think I figured it out - pushed up some changes to pass linting checks...

Reply:
Hm, looks like not everything here was linted. Sorry! Maybe it would be easier to run `black .` from the root of the repo?

Reply:
ok - just did that, and pushed back up...  Thanks!